server/utils/express.js

```diff
@@ -12,16 +12,37 @@ const logger = config.logger('express');
  * Ensures that the specified object is an Express request, or throws an error.
  *
  * @param {Request} req - An Express request object.
+ * @param {string} description - An optional description of where the object comes from (e.g. first argument).
  * @returns {Request} The first argument, unchanged.
  */
-exports.ensureRequest = function(req) {
-  if (!req || !req.app) {
-    throw new Error('Argument must be an Express request');
+exports.ensureRequest = function(req, description = 'Argument') {
```

Comment:
I don't understand why you added this param.
I understand the intent, but it doesn't seem to be used anywhere.